app/scripts/lib/verification-reasons.js

```diff
@@ -12,6 +12,7 @@ define(function (require, exports, module) {
   return {
     FORCE_AUTH: 'force_auth',
     PASSWORD_RESET: 'reset_password',
+    PASSWORD_RESET_WITH_RECOVERY_KEY: 'reset_password_with_recovery_key',
```

Comment:
Added a specific view for when a user completes resetting their password with recovery key. This was used to differentiate it in metrics from a regular reset.